requirements.txt

```diff
@@ -1,7 +1,16 @@
+## The following requirements were added by pip freeze:
```

Comment:
should we split the file into `req...` and `req..._dev`? I don't know whats the best practice here.

Reply:
I splitted the files (220decac1c7c3f11eb88c8c404ab095ec8cbd8ee)

Reply:
Actually, best practice is having a `requirements.txt` for dev and a `setup.py` for production. However, I like simple pip requirements so much more myself. So, I am fine with having two files. Just keep in mind to keep them up-to-date, both.